### DISCUSSION TITLE
error when building with static adapter + mdsvex

### DISCUSSION THREAD

**Discussion by brunoti** [sveltejs/kit#11150] [Q&A] [answered]
I'm receiving this error when building

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/bruno/personal/personal-website/.svelte-kit/output/server/entries/posts/adding-dependencies-on-clojure-projects-th
e-node-way-a-small-intro-to-neil-cli.md'
```

You can find the code here https://github.com/brunoti/blog. Clone, install, build: you will receive the error.

I've seen some other examples and could not find why mine doesn't work.

**Comment by teemingc** [comment]
It's erroring from this import in the `+page.js` file. You are trying to import a path that doesn't exist during the prerender process. 
https://github.com/brunoti/blog/blob/102efe35b04bcaf6b5623bbb791af0e34ba909e8/src/routes/writing/%5Bslug%5D/%2Bpage.ts#L7-L10

Where did you find this example? It doesn't seem like it would work under any circumstances.

**Reply by brunoti** [reply]
It works on dev server. So it actually exists. Also tried making the filer a page.server only also didn’t work.

There is this one and a joy of code one also. They both builds ok.

https://github.com/josh-collinsworth/sveltekit-blog-starter

**Reply by brunoti** [reply, marked as answer]
Could be the vite-ignore comment? I thought it would only make vite don’t warn about dynamic imports.

**Reply by brunoti** [reply]
Yes, it was!

That comment deceived me. I need to read about it and see why it causes it.

**Reply by brunoti** [reply]
Thanks anyway!
